src/routes/welcome/gettingStarted.tsx

```diff
@@ -0,0 +1,178 @@
+import { AnimatePresence, type Variants, motion } from "framer-motion";
+import { useCallback, useState } from "react";
+import { Button, Card } from "@arconnect/components";
+import { CheckIcon } from "@iconicicons/react";
+import browser from "webextension-polyfill";
+
+import styled from "styled-components";
+
+import Completed from "./gettingStarted/completed";
+import Connect from "./gettingStarted/connect";
+import { useLocation } from "wouter";
+import Explore from "./gettingStarted/explore";
+
+const gettingStartedPages = [<Completed />, <Explore />, <Connect />];
+
+export default function GettingStarted({ page }) {
+  // animate content sice
+  const [contentSize, setContentSize] = useState<number>(0);
+  const [, setLocation] = useLocation();
+  const contentRef = useCallback<(el: HTMLDivElement) => void>((el) => {
+    if (!el) return;
+
+    const obs = new ResizeObserver(() => {
+      if (!el || el.clientHeight <= 0) return;
+      setContentSize(el.clientHeight);
+    });
+
+    obs.observe(el);
+  }, []);
+
+  const navigate = (pageNum: number) => {
+    if (pageNum < 4) {
+      setLocation(`/getting-started/${pageNum}`);
+    } else {
+      window.top.close();
+    }
+  };
+
+  return (
+    <Wrapper>
+      <SetupCard>
+        <HeaderContainer>
+          <Header>
+            <CheckWrapper>
+              <CheckIcon />
+            </CheckWrapper>
+            Setup Complete!
+          </Header>
+        </HeaderContainer>
+        <Content>
+          <PageWrapper style={{ height: contentSize }}>
+            <AnimatePresence initial={false}>
+              <Page key={page} ref={contentRef}>
+                {gettingStartedPages[page - 1]}
+              </Page>
+            </AnimatePresence>
+          </PageWrapper>
+        </Content>
+        <Footer>
+          <PageIndicatorContainer>
+            {gettingStartedPages.map((_, i) => (
+              <PageIndicator
```

Comment:
The white bg for the other page indicators looks off. Let's change that to something that is consistent with ArConnect's style. Maybe `rgba(theme_color, .3)`, we use that for secondary bgs. Like so:
<img width="148" alt="Screenshot 2023-10-20 at 7 34 19 PM" src="https://github.com/arconnectio/ArConnect/assets/30638105/1e423cd1-e03a-4424-a839-af3cb9a11306">